github/messages.go

```diff
@@ -30,6 +31,35 @@ const (
 	sha512Prefix = "sha512"
 	// signatureHeader is the GitHub header key used to pass the HMAC hexdigest.
 	signatureHeader = "X-Hub-Signature"
+	// eventTypeHeader is the Github header key used to pass the event type
```

Comment:
Minor style nitpick. This sentence is missing a period at the end, which is inconsistent with the Go style and this codebase.